openmetadata-service/src/main/java/org/openmetadata/service/resources/apps/AppResource.java

```diff
@@ -887,6 +887,10 @@ public Response triggerApplicationRun(
           String name) {
     EntityUtil.Fields fields = getFields(String.format("%s,bot,pipelines", FIELD_OWNER));
     App app = repository.getByName(uriInfo, name, fields);
+    if (Boolean.TRUE.equals(app.getSystem())) {
```

Comment:
wouldnn't this stop triggering SeachIndexApp or DataInsights

Reply:
no. they are not defined as system apps:
https://github.com/open-metadata/openmetadata-collate/blob/9d92d18944441e0db5b5b2aee9e6fc7780653e1c/collate-service/src/main/resources/json/data/appMarketPlaceDefinition/telemetryApplication.json#L21
https://github.com/open-metadata/OpenMetadata/blob/173076b55e4877335bfdbccdcdf069f80a49fee0/openmetadata-service/src/main/resources/json/data/appMarketPlaceDefinition/SearchIndexingApplication.json